Xplat/src/main/java/vazkii/botania/common/block/block_entity/CellularBlockEntity.java

```diff
@@ -67,8 +67,8 @@ public void update(Level level) {
 		generation = nextGeneration;
 	}
 
-	public boolean hasPoweredParent(Level level) {
-		return flowerCoords != null && level.getBlockEntity(flowerCoords) instanceof DandelifeonBlockEntity && level.hasNeighborSignal(flowerCoords);
+	public boolean hasActiveParent(Level level) {
+		return flowerCoords != null && level.getBlockEntity(flowerCoords) instanceof DandelifeonBlockEntity parent && level.hasNeighborSignal(flowerCoords) && ((level.getGameTime() % DandelifeonBlockEntity.SPEED == 0) || parent.overgrowth);
```

Comment:
I'm not sure where else it would go, as putting in a different method would require duplicating the calls to verify that flowerCoords points to a real dandelifeon. I've removed the timing calls however.